content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx

```diff
@@ -8,23 +8,21 @@ preview: true
 
 <TopBlock>
 
-Prisma allows configuration of database indexes, unique constraints and primary key constraints through the `extendedIndexes` preview feature in version `3.5.0` and later. This provides extended syntax options in the Prisma schema language for configuring the length of indexes (MySQL only), the sort order (all databases), the index type (PostgreSQL only) and to configure indexes as clustered or non-clustered (SQL Server only).
+Prisma allows configuration of database indexes, unique constraints and primary key constraints. This is in General Availability in versions `4.0.0` and later. You can enable this with the `extendedIndexes` Preview feature in versions `3.5.0` and later.
 
-Version `3.6.0` also introduces support for introspection and migration of full text indexes in MySQL and MongoDB through a new `@@fulltext` attribute, available through the `fullTextIndex` preview feature.
+Version `3.6.0` also introduces support for introspection and migration of full text indexes in MySQL and MongoDB through a new `@@fulltext` attribute, available through the `fullTextIndex` Preview feature.
 
 <Admonition type="warning">
 
-These changes to index configuration and full text indexes might be **breaking changes** if you have a database that is already making use of these features, so they need to be explicitly enabled via preview features.
+These changes to index configuration and full text indexes might be **breaking changes** if you have a database that already uses these features.
 
 </Admonition>
 
 </TopBlock>
 
 ## Index configuration
 
-The `extendedIndexes` preview feature enables new configuration options for indexes, unique constraints and primary key constraints in Prisma.
-
-The following arguments can be specified:
+You can configure indexes, unique constraints, and primary key constraints with the following attribute arguments:
 
 - The `length` argument is available **in MySQL only** on the `@id`, `@@id`, `@unique`, `@@unique` and `@@index` attributes in version `3.5.0` and later. It allows Prisma to support indexes and constraints on `String` and `Bytes` types.
```

Comment:
I'd love to not have the versions from when this was available in preview in here. Would it be fine to say available from 4.0?

Reply:
Also, it's important to note that for this line here, it is available from `3.5.0` but the preview feature flag was also required.

Reply:
We usually go all the way here indeed an spell out since when it was GA available, but then also add as an aside with which preview feature it was availalbe for which versions.

Example: https://www.prisma.io/docs/concepts/components/prisma-schema/relations/referential-actions#what-are-referential-actions:~:text=In%202.26.0%20and%20later%20(before%203.0.0%20with%20a%20preview%20feature%20flag%20referentialActions%2C%20afterwards%20without)%2C%20you%20can%20define%20referential%20actions%20on%20your%20relation%20fields%20in%20your%20Prisma%20schema.

This is useful as we have and will keep having loads of users on versions between 3.5.0 and 4.0.0 where the preview feature is still required.

(We have no policy yet when and if we remove these outdated preview notes)

Reply:
(In this case it might make sense to just seaprate the preview inforamtion into a section _after_ the initial section so that this becomes easier to read as @do4gr suggested. The preview information could go in stead of where we currentl have the instrucions on how to enable the preview feature then.)

Reply:
Agree that this could do with reorganisation, made an issue to look at this after the release: https://github.com/prisma/docs/issues/3332